x-pack/plugins/security_solution/public/detections/components/alerts_kpis/alerts_count_panel/alerts_count.tsx

```diff
@@ -24,7 +24,7 @@ interface AlertsCountProps {
 }
 
 const Wrapper = styled.div`
-  overflow: scroll;
+  overflow: auto;
```

Comment:
This isn't needed if you're using the css helpers. Also super curious why the negative margin in the line below is needed. If it is truely needed it should probably use the correct EUI size token rather than a pixel value.

Reply:
#### Why negative margin? 
`EuiBasicTable` has `8px` padding around every cell. Because the alert count table has no border around cells, the 8px padding on the top of the table header gets visually combined with the `24px` margin-bottom of the section header.  

<img width="500" alt="Screenshot 2021-08-19 at 12 34 23" src="https://user-images.githubusercontent.com/1490444/130054461-52e3f1b5-e300-4147-afb8-d0b1ae66eb45.png">

<img width="500" alt="Screenshot 2021-08-19 at 12 15 01" src="https://user-images.githubusercontent.com/1490444/130054003-5df117fb-42aa-4341-a738-370a68278a08.png">

To compensate for this combination of margin and padding, I defined a negative margin to the table, so the distance between the section header and the table header is truly`24px`.

<img width="500" alt="Screenshot 2021-08-19 at 12 14 07" src="https://user-images.githubusercontent.com/1490444/130054193-3960f41a-0cc2-4fc0-8373-a76c9d350141.png">